package.json

```diff
@@ -211,6 +211,7 @@
     "@typescript-eslint/eslint-plugin": "^7.3.1",
     "@typescript-eslint/parser": "^7.3.1",
     "@vscode/l10n-dev": "^0.0.24",
+    "@vscode/test-electron": "^2.3.8",
```

Comment:
Moved it now to the main package.json since we lint this folder now. Which means we need test-electron types. I checked if there is any post-install or install step and there isn't, so I don't see any reason to not put it into devDependencies.